src/XMakeBuildEngine/Construction/ProjectMetadataElement.cs

```diff
@@ -49,7 +49,14 @@ public string Name
         }
 
         //  Add a new property with the same name here because this attribute should be public for ProjectMetadataElement,
-        //  but internal for ProjectElement, because we don't want it to be settable for arbintrary elements.
+        //  but internal for ProjectElement, because we don't want it to be settable for arbitrary elements.
```

Comment:
It's talking about internal APIs, so I don't think it goes in the public API documentation that comes from the Xml comments